player/public/coreplugins/org.ekstep.videorenderer-1.1/renderer/plugin.js

```diff
@@ -190,7 +192,11 @@ org.ekstep.contentrenderer.baseLauncher.extend({
             }
         });
         videojs("videoElement").ready(function () {
-			var youtubeInstance = this;
+            var youtubeInstance = this;
+            var iframeSrc = $('#videoElement iframe')[0].src;
+            if(window.cordova && globalContext.origin){
+                $('#videoElement iframe')[0].src = instance.replaceOrigin(iframeSrc, "origin", globalContext.origin)
```

Comment:
var origin = EkstepRendererAPI.getGlobalConfig().origin;
if(origin) {
  var iframeSrc = $('#videoElement iframe')[0].src;
  $('#videoElement iframe')[0].src = instance.replaceOrigin(iframeSrc, "origin", origin)
}

Reply:
As we are replacing origin only for mobile play, we need to check the Cordova check there for mobile.